app/controllers/api/zones_controller.rb

```diff
@@ -1,6 +1,9 @@
 module Api
   class ZonesController < BaseController
+    include Subcollections::Authentications
+
     INVALID_ZONES_ATTRS = ID_ATTRS + %w[created_on updated_on].freeze
+    VALID_AUTH_TYPES    = %w[windows_domain].freeze
```

Comment:
should be made available to the API via app/models/zone.rb

Reply:
You mean, like the "possible" values?  I don't think so, because we don't do that for other things.  This specfic value, currently, would not be used in anything like a drop down, so there would be no caller for that kind of method.

Reply:
Interesting, even though "windows_domain" is a value we look for in host and vms ?  (just doing a git grep windows_domain). You'd think it's a model declared list (valid auth types).

Reply:
Oh sorry...I misunderstood your request.  I thought you were stating you wanted it exposed over the API as opposed to what you really meant which was declaring it in the model.  On that I agree...this list should be in the model  See also my comment: https://github.com/ManageIQ/manageiq-api/pull/916#discussion_r502706666

Reply:
As per 

> I am _kind of_ ok with hardcoding in api/zones_controller.rb as that's where the hardcoding waas previously in the UI controller

I'd rather it ultimately made it into the Zone model, but I'd be ok with a follow-up PR.

Reply:
👍

Reply:
> I'd rather it ultimately made it into the Zone model, but I'd be ok with a follow-up PR.

I agree with this as well.

I think a follow up makes the most sense at this point just so we can get something out to the UI team, and then fix in a follow up (which will require a little more coordination than a single PR and merge)